sm_cnn/README.md

```diff
@@ -144,7 +96,7 @@ NB: The results on WikiQA are based on the SM model hyperparameters.
 to the `data/` folder
 
 ```bash
-python $PYTHONPATH/utils/build_w2v.py --input data/aquaint+wiki.txt.gz.ndim=50.bin
+python $PYTHONPATH/utils/build_w2v.py --input ../../Castor-data/word2vec/embeddings/aquaint+wiki.txt.gz.ndim=50.bin
```

Comment:
different from the path before `embeddings/word2vec`?

Reply:
Yeah, this is the correct behaviour. We moved `GloVe`, `word2vec` etc. to `embeddings` per @lintool's suggestion.